src/index.ts

```diff
@@ -58,14 +58,19 @@ const defaultOptions: Partial<PinoSentryOptions> = {
   expectPinoConfig: false,
 };
 
+const defaultSentryOptions: NodeOptions = {
+  dsn: process.env.SENTRY_DSN,
+  environment: process.env.NODE_ENV,
```

Comment:
Sentry environment is not 1:1 to NODE_ENV.
There can be different types of production environments like staging which should be NODE_ENV=production because different library do optimization when NODE_ENV=production.

Reply:
We migrated from Winston to Pino. And was expecting a similar configuration. But your comments make sense. I'll close the PR. But perhaps it makes sense to update your lock file ;)

Reply:
@pdeveltere thanks!